src/runtime/composables/useSession.ts

```diff
@@ -33,6 +33,8 @@ interface SignInOptions extends Record<string, unknown> {
   callbackUrl?: string
   /** [Documentation](https://next-auth.js.org/getting-started/client#using-the-redirect-false-option) */
   redirect?: boolean
+
+  replace?: boolean
```

Comment:
I propose to either:
- not make this an option for the lib-user (downsides are unclear naming + we'd have to replicate the behavior for `useSession` as well, as it has the same options as `signIn`)
- also add this to `useSession` options + behavior



I slightly prefer option 1 (:

Reply:
I would go with option 2, as we should not use `replace` by default when the user calls signIn(), only in auth middleware. And if the user want to create a custom auth middleware, he can enable it.